python/lsst/jointcal/utils.py

```diff
@@ -366,9 +366,8 @@ def get_fluxes(photoCalib, match):
                 else:
                     ref_flux = refcalib.instFluxToNanojansky(match[0], ref_flux_key).value
             else:
-                # refcat fluxes are already in Janskys.
-                # TODO: Once RFC-549 is fully implemented, we can remove the 1e9 prefactor.
-                ref_flux = 1e9 * match[0][ref_flux_key.format(filt)]
+                # refcat fluxes are already in nanojansky.
```

Comment:
I'm not sure I understand why the flux would be in jansky if it is in a slot, but nanojansky otherwise.

Reply:
The `slot` vs. no-`slot` difference here is how I tell whether I the match is against a refcat (no-slot, fluxes in nJy) or SourceCat (`slot`, instFluxes in counts that have to be converted to nJy with a PhotoCalib).